functions/account.yml

```diff
@@ -0,0 +1,30 @@
+createAccount:
+  handler: src/handler/account-handler.createAccount
+  events:
+    - http:
+        path: accounts/create
```

Comment:
path should be just "accounts", without "/create"

Reply:
Changed in a previous PR by Marko.